docs/source/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.rst

```diff
@@ -42,87 +42,194 @@ Antenatal care attendance module
+    - Propensity value
+    - Notes
+  * - A
+    - Attends ANC during first trimester AND later pregnancy
+    - 0
+    - Assumption
+  * - B
+    - Attends ANC during first trimester but NOT later pregnancy
+    - ``ANCfirst``
+    -
+  * - C
+    - Attends ANC during later pregnancy but NOT first trimester
+    - 0 
+    - Assumption
+  * - D
+    - Does not attend ANC at all during pregnancy
+    - Probability equal to ``1 – ANCfirst``  
+    - 
 
-2.4 Module Action Points
----------------------------
 
-.. list-table:: Module action point
+2.2 Module Inputs
+-----------------
+
+.. list-table:: Module required inputs
   :header-rows: 1
 
-  * - Action point
-    - Description
-    - Information
+  * - Input
+    - Source 
+    - Application
     - Note
-  * - I
-    - Record ANC attendance
-    - Record to output A
+  * - Pregnancy term duration 
+    - :ref:`Pregnancy I <2024_vivarium_mncnh_portfolio_pregnancy_module>`
+    - Partial term pregnancies by default should be assigned the probability value of ANCfirst.
+    - 
+  * - ANCfirst
+    - ANCfirst is processed by the Health Systems team at IHME and available here:
+      ``J:\Project\simulation_science\mnch_grant\MNCNH portfolio\anc1_first3months_st-gpr_results_aggregates_scaled2025-05-27.csv``
+    - Used to calculate propensity values for ANC coverage
+    - 
+  * - ANC1
+    - GBD covariate ID 7: :code:`get_covariate_estimates(location_id=location_id, release_id=16, year_id=2023, covariate_id=7)` 
+    - Used to calculate propensity values for ANC coverage
     - 
+  * - ANC4
+    - GBD covariate ID 8: :code:`get_covariate_estimates(location_id=location_id, release_id=16, year_id=2023, covariate_id=8)` 
+    - Used to calculate propensity values for ANC coverage
+    - 
+
+2.3 Module Outputs
+++++++++++++++++++
+
+As mentioned earlier, ANC attendance impacts hemoglobin exposure and facility choice in our model, and in order for the 
+outputs of this component to be compatible with the data needs of these two downstream components, we will need two different
+outputs, one being dichotomous for the hemoglobin component and the other being polychotomous for the facility choice component. 
 
-2.4: Module Outputs
------------------------
+ANC attendance inputs to the :ref:`hemoglobin module <2024_vivarium_mncnh_portfolio_hemoglobin_module>`
+are dichotomous for each pregnancy, so we need to observe the following: 
 
-.. list-table:: Module outputs
+.. list-table:: Module outputs for hemoglobin component
   :header-rows: 1
 
   * - Output
     - Value
     - Note
-  * - A. ANC1?
+  * - Attends ANC in first trimester?
     - *True* / *False*
-    - 
+    - For groups B or D 
+  * - Attends ANC in later pregnancy?
+    - *True* / *False* 
+    - For groups A or C
```

Comment:
Similarly, what does "For groups A or C" mean? I think this one should be True for groups A and C, and False for groups B and D.